src/action.js

```diff
@@ -131,7 +131,7 @@ function cropRangeList(separator, showMissingMaxLength, ranges) {
 }
 
 function linkRange(fileUrl, range) {
-  const [start, end] = range.slice(1, -1).split("-", 2);
+  const [start, end] = range.split('&NoBreak;').join().slice(1, -1).split("-", 2);
```

Comment:
I messed this one up too. The join method adds a comma

```suggestion
  const [start, end] = range.replace('&NoBreak;','').slice(1, -1).split("-", 2);
```